src/prog_models/models/centrifugal_pump.py

```diff
@@ -121,79 +122,92 @@ class CentrifugalPump(prognostics_model.PrognosticsModel):
             'To': 350
         },
 
+        # Wear Rates
+        'wA': 0,
+        'wRadial': 0,
+        'wThrust': 0,
+
         # Initial state
         'x0': {
-            'w': 376.991118431, # 3600 rpm (rad/sec)
+            'w': 376.991118431,  # 3600 rpm (rad/sec)
             'Q': 0,
             'Tt': 290,
             'Tr': 290,
             'To': 290,
             'A': 12.7084,
             'rThrust': 1.4e-6,
             'rRadial': 1.8e-6,
-            'wA': 0.0,
-            'wThrust': 0,
-            'wRadial': 0,
         }
     }
 
+    state_limits = {
+        'To': (-273.15, inf),
+        'Tr': (-273.15, inf),
+        'Tt': (-273.15, inf),
```

Comment:
The state limits here seem like they should be in Kelvin to agree with the rest of the model (e.g. 0 instead of -273.15). If the intention is to have everything in the model be in Celsius, then x0 should probably be different (e.g. x0['To'] should be 16.85 instead of 290, along with the two other temperatures). Because the differential equations are only impacted by differences in temperatures, using Celsius as opposed to Kelvin shouldn't have an impact.